<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="TB_SELECT">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>蓝牙设备</string>
       </attribute>
       <widget class="QPushButton" name="PB_SEARCHBLT">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>430</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>搜索设备</string>
        </property>
       </widget>
       <widget class="QLabel" name="LB_CONNECTSTAT">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>440</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string> 已连接:无</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PB_BLTBREAK">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>430</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LE_BLTSEND">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>430</y>
          <width>271</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="PB_BLTSEND">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>420</y>
          <width>111</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>301</width>
          <height>401</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string> 已配对</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="SA_BLUEDEVPAIRED">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>297</width>
              <height>172</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QWidget" name="WG_BLUEDEV_2" native="true"/>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>397</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string> 未配对</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="SA_BLUEDEV">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>297</width>
              <height>172</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QWidget" name="WG_BLUEDEV" native="true"/>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>397</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QComboBox" name="CB_BLTTEXTMODE_SEND">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>430</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="TB_BLTRECV">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>50</y>
          <width>321</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
       <widget class="QScrollArea" name="SA_BLTOBJGROUP">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>50</y>
          <width>271</width>
          <height>361</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>269</width>
           <height>359</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QWidget" name="widget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>317</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>接收区</string>
        </property>
       </widget>
       <widget class="QComboBox" name="CB_BLTAPPEND">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>460</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>14</y>
          <width>72</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> 控件组</string>
        </property>
       </widget>
       <widget class="QComboBox" name="CB_BLTTEXTMODE_RECV">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="TB_BLUETOOTH">
       <attribute name="title">
        <string>串口设备</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>-190</x>
          <y>0</y>
          <width>1311</width>
          <height>461</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>数据流控制</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="CB_FLOWCTRL"/>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QPushButton" name="PB_CONNECT_COM">
             <property name="text">
              <string>连接</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QTextEdit" name="TE_SENDTEXT_COM"/>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="CB_COM"/>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="CB_STOPBIT"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>波特率</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>校验位</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="CB_BOTERATE"/>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="CB_ENSUREBIT"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>停止位</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QPushButton" name="PB_SENDTEXT_COM">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string> 端口  </string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>数据位</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="CB_DATABITS"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="TB_RECVTEXT_COM"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2"/>
         </item>
         <item>
          <widget class="QScrollArea" name="SA_VARS_COM">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>186</width>
              <height>457</height>
             </rect>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>221</width>
               <height>551</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QWidget" name="WG_VARS_COM" native="true"/>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="TB_TCP">
       <attribute name="title">
        <string>TCP 设备</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
